.circleci/config.yml

```diff
@@ -245,3 +365,8 @@ workflows:
             - test_PE_unstranded_singularity
             - test_SE_unstranded_singularity
             - test_PE_stranded_singularity
+            - test_PE_align_docker
+            - test_SE_align_docker
+            - test_SE_align_singularity
+            - test_PE_kallisto_docker
```

Comment:
Do you need test_PE_align_singularity here?

Reply:
concatenation is baked in into the align and kallisto steps. The added task level test for kallisto and align address the concatenation.

Reply:
test_PE_align_singularity is needed, thank you for pointing this out.